Monika-After-Story/MonikaModDev
# Scary stories

Now Monika can tell you some scary stories ~~and try to scare you the same way ddlc did~~.
This one adds 12 new scary stories that Monika can tell the player.

## Changes
- There's going to be a new topic `Can you tell me a horror story?` that's going to trigger the flow for the scary stories. First time she'll jump straight into the first story, consecutive times she'll ask the player to pick from a menu the story they want to hear.
- There's a new category `TYPE_SCARY` for stories.
- There's a new script that contains some of the definitions for the effects used on some of the stories, it's called `special-effects.rpy`
- There's no scary stories for sensitive mode since these include content `not suitable for children\nor those who are easily disturbed.`
- Adds a function to `mas_stories` store that unlocks every story for easy testing. Function is `_unlock_everything()`. 

## Testing
- Scary stories unlock the same way that normal stories do, one each session.
- Set `mas_full_scares` to True, to see the scares that some stories have.
- Check that the topic for the stories isn't available on sensitive mode.
- Check that the stories correctly restore the game state after the story ends.
- If you're no longer testing the unlock flow you can execute on the console `mas_stories._unlock_everything()` to unlock every story.


### Special Thanks
- @kaido1224 for the stories and general suggestions.
- @sebastianN01 for the art.
- @multimokia for the scary soundtrack.
